styles/style.css

```diff
@@ -18,25 +21,63 @@
   margin: 0 auto;
 }
 
-.section-icon {
+.container {
+  display: flex;
+  justify-content: center;
   max-width: 1440px;
   box-sizing: border-box;
-  margin: 0 auto;
   padding: 80px 88px 0 104px;
 }
 
 .icon {
-  display: inline-block;
   width: 296px;
   height: 176px;
   text-align: center;
-  float: left;
   margin-right: 16px;
   margin-bottom: 122px;
 }
 
 .caption {
   font-family: 'Roboto', sans-serif;
   font-weight: 400;
+  font-size: 20px;
   margin-top: 48px;
+  line-height: 28px;
+}
+
+/* footer */
+.footer {
+  background-color: #10367C;
+  color: #ffffff;
+}
+
+.container-footer {
+  display: flex;
+  justify-content: space-between;
+  padding: 120px 272px 64px 104px;
+}
+
+.footer-list {
+  list-style: none;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 24px;
 }
+
+li{
+  margin-bottom: 24px;
+}
+
+ul li:first-child {
+  font-weight: 500;
+}
+
+.copyright {
```

Comment:
<img width="1432" alt="Screenshot 2022-03-05 at 18 15 23" src="https://user-images.githubusercontent.com/25488411/156889425-044bfbd4-172b-4ba6-bbd2-d16baa94ecf7.png">
The copyright is too low, try to undestend why and fix. If you don't guess, message me